script.js

```diff
@@ -0,0 +1,139 @@
+function getCourseCard(course){
+    let html = 
+    `
+    <a href="#" class="course-card">
+                    <img src="${course?.image??"assets/PLMP"}" alt="courseImg" width="100%">
+                    <div class="course-text">
+                        <h3>${course?.title??"Course Name"}</h3><span class="instructorName">${course?.instructor??"Jeff Bezos"}</span>
+                        <div>
+                            <span class="rating">${course?.rating??4.5}</span>
+                            ${course.rating>=1?`<span class="fa fa-star checked"></span>`:`<span class="fa fa-star"></span>`}
+                            ${course.rating>=2?`<span class="fa fa-star checked"></span>`:`<span class="fa fa-star"></span>`}
+                            ${course.rating>=3?`<span class="fa fa-star checked"></span>`:`<span class="fa fa-star"></span>`}
+                            ${course.rating>=4?`<span class="fa fa-star checked"></span>`:`<span class="fa fa-star"></span>`}
+                            ${course.rating>=5?`<span class="fa fa-star checked"></span>`:`<span class="fa fa-star"></span>`}
+                        </div>
+                        <h3>${course?.price??"Free"}</h3>
+                    </div>
+                </a>
+    `
+    return html;
+}
+function createCourse(course) {
+    let html = 
+        `
+            <div class="course carousel-item">
+                ${getCourseCard(course)}
+            </div>
+            `;
+    return html;
+}
+let currentTab = "Python";
+function updateCourses(courses){
+    console.log("hi how are you?");
+    let coursesContainer = document.querySelector("#myTabContent");
+    courses.forEach((course) => coursesContainer.innerHTML += getCourseCard(course));
```

Comment:
Search is working! But the styles are not. Let's have a small meeting to work on it together

Reply:
Ok, will the meeting be during the group meeting?

Reply:
Can do it right now, check slack